packages/pass-style/test/passStyleOf.test.js

```diff
@@ -193,16 +197,27 @@ test('passStyleOf testing remotables', t => {
 
   const tagRecord1 = harden(makeTagishRecord('Alleged: manually constructed'));
   /** @type {any} UNTIL https://github.com/microsoft/TypeScript/issues/38385 */
-  const farObj1 = harden({
-    __proto__: tagRecord1,
-  });
+  // TODO Once we have support for explicit non-trapping, this should change
+  // from `harden` to `suppressTrapping` or `stabilize`.
+  const farObj1 = harden({ __proto__: tagRecord1 });
```

Comment:
Please remove the `TODO` noise?

```suggestion
  const farObj1 = suppressTrapping({ __proto__: tagRecord1 });
```

... and so on for every `suppressTrapping` TODO.

Reply:
Done